src/Skoruba.IdentityServer4.Admin/Helpers/TagHelpers/PickerTagHelper.cs

```diff
@@ -78,12 +62,7 @@ private void AddComponent(TagHelperOutput output)
         {
             var selectedItems = GetSelectedItemsToJsonString();
 
-            var rawPickerHtml = new HtmlString(
-                "<div data-bind='component: { name: \"picker\", params: { search: \"\", hiddenId: \"" + Id + "\", url: \"" +
-                Url + "\", selectedItemsTitle: \"" + SelectedItemsTitle + "\", allowSuggestedItems: " + AllowSuggestedItems.ToString().ToLower() + ", searchResultTitle: \"" +
-                SearchResultTitle + "\", suggestedItemsTitle: \"" + SuggestedItemsTitle + "\", noItemSelectedTitle: \"" +
-                NoItemSelectedTitle + "\", searchInputPlaceholder: \"" + SearchInputPlaceholder + "\", showAllItemsTitle: \"" + ShowAllItemsTitle + "\", selectedItems: " + selectedItems + ", minSearchText: " + MinSearchText +
-                ", topSuggestedItems: " + TopSuggestedItems + " ,multipleSelect: " + MultipleSelect.ToString().ToLower() + " }}'></div>");
+            var rawPickerHtml = new HtmlString($"<div data-bind=\'component: {{ name: \"picker\", params: {{ search: \"\", hiddenId: \"{Id}\", url: \"{Url}\", selectedItemsTitle: \"{SelectedItemsTitle}\", allowSuggestedItems: {AllowSuggestedItems.ToString().ToLower()}, searchResultTitle: \"{SearchResultTitle}\", suggestedItemsTitle: \"{SuggestedItemsTitle}\", noItemSelectedTitle: \"{NoItemSelectedTitle}\", searchInputPlaceholder: \"{SearchInputPlaceholder}\", showAllItemsTitle: \"{ShowAllItemsTitle}\", selectedItems: {selectedItems}, minSearchText: {MinSearchText}, topSuggestedItems: {TopSuggestedItems} ,multipleSelect: {MultipleSelect.ToString().ToLower()} }}}}\'></div>");
```

Comment:
You don't need to escape the `'` here around `data-bind` which will make the verbatim string a bit shorter.

Reply:
Maybe something like this would work an would be more readable:

```cs
var component = new {
	name = "picker",
	@params = new {
		search = "",
		hiddenId = Id,
		url = Url,
		selectedItemsTitle = SelectedItemsTitle,
		allowSuggestedItems = AllowSuggestedItems.ToString().ToLower(),
		searchResultTitle = SearchResultTitle,
		suggestedItemsTitle = SuggestedItemsTitle,
		noItemSelectedTitle = NoItemSelectedTitle,
		searchInputPlaceholder = SearchInputPlaceholder,
		showAllItemsTitle = ShowAllItemsTitle,
		selectedItems = selectedItems, // Should this really be lowercase?
		minSearchText = MinSearchText,
		topSuggestedItems = TopSuggestedItems,
		multipleSelect = MultipleSelect.ToString().ToLower()
	}
};
$"<div data-bind='component: {JsonConvert.SerializeObject(component)}'>"
```